xmodule/library_content_block.py

```diff
@@ -436,15 +504,11 @@ def author_view(self, context):
                 max_count = self.max_count
                 if max_count < 0:
                     max_count = len(self.children)
-
-                fragment.add_content(self.runtime.service(self, 'mako').render_cms_template(
-                    "library-block-author-preview-header.html", {
-                        'max_count': max_count,
-                        'display_name': self.display_name or self.url_name,
-                    }))
+                context = {} if not context else copy(context)
                 context['can_edit_visibility'] = False
                 context['can_move'] = False
                 context['can_collapse'] = True
+                context['selectable'] = True
```

Comment:
We should probably leave `selectable = False` when `manual==False`, do you agree?